cmd/machine-controller/Makefile

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 The Kubernetes Authors.
```

Comment:
We don't have this Makefile anymore and one located in top level directory already can do this.

Reply:
Yeah, I will update the JJ. This was just needed to make it run :).

Reply:
Removed